### ISSUE TITLE
[Suggestion] Allow levels to import their own music into levels for added experience

### ISSUE THREAD

**Issue by TidesSonic** [whitelilydragon/ShangMuArchitect#259]
I'd completely love to have the ability to import custom music into a level to give the level it's own unique feel compared to other levels.

**Comment by whitelilydragon** [comment]
This is something that could be technically feasible to do, but it creates too many potential legal issues, because of how the music industry works.

I don't want Freedom Planet fans making custom stages to face takedown requests of their maps, or recordings of their maps on YouTube, because of songs they used for their stages. That could especially scare a lot of young teens who are unaware of how much tighter the music industry is about the distribution and "broadcasting" of intellectual property.

Once the game is complete, there should be a healthy enough variety of music to create the moods people want. I know that won't carry the same sentiment as using your favorite song, but it would burden my conscious to let people run afoul of the above legal problems.

So, I must label this Won't Fix, I apologize.
